skimage/morphology/_util.py

```diff
@@ -3,9 +3,14 @@
 
 import numpy as np
 from scipy import ndimage as ndi
-
-
-def _validate_connectivity(image_dim, connectivity, offset):
+from typing import Union, Optional, Tuple, Sequence
+from ..typing import _SupportsArray
+
+def _validate_connectivity(
+        image_dim: int,
+        connectivity: Optional[Union[int, _SupportsArray]],
+        offset: Optional[Sequence[int]],
```

Comment:
```suggestion
    image_dim: int,
    connectivity: Optional[Union[int, _SupportsArray]],
    offset: Optional[Sequence[int]],
```
I'd be interested in your reasoning about adding two indent levels and deviating from the "norm" here? :wink:

Reply:
@lagru "the norm" is a choice that black made that I dislike. PEP8 encourages two levels to distinguish the arguments from the body of the function: https://www.python.org/dev/peps/pep-0008/#indentation